Bottom: 400e95a23e7148ec9a2269cd7d2866d99bdbd49e
Top:    a4b56ad114402e94fea42cb4c283ab16ccc8009d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 11:31:49 +0100

hid/png: Increase the maximum allowable DPI of the exporter

Sometimes for rendering fiddly boards, you just need the detail.

Move the arbitrary 1000 dpi limit to still arbitrary, 10000 dpi.


---

diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index 86631a7..bfa98ad 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -169,7 +169,7 @@ HID_Attribute png_attribute_list[] = {
 #define HA_pngfile 0
 
   {"dpi", "Scale factor (pixels/inch). 0 to scale to fix specified size",
-   HID_Integer, 0, 1000, {100, 0, 0}, 0, 0},
+   HID_Integer, 0, 10000, {100, 0, 0}, 0, 0},
 #define HA_dpi 1
 
   {"x-max", "Maximum width (pixels).  0 to not constrain.",
